.github/workflows/publish.yml

```diff
@@ -1,4 +1,4 @@
-# TFS-API Git Hub Actions
+# MDM-API Git Hub Actions
```

Comment:
Copy paste from other project.

This PR has 47 commits, but should be fine if squashed.